scripts/releasing.gradle

```diff
@@ -0,0 +1,61 @@
+apply plugin: 'maven-publish'
+
+task sourceJar(type: Jar) {
+  from sourceSets.main.allJava
+  classifier 'sources'
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+  from javadoc.destinationDir
+  classifier = 'javadoc'
+}
+
+// this is the only way for us to create a pom using the gradle tools
+// usually this same config is used to deploy, but we're deploying
+// using another tool and really just want the pom.
+publishing {
+  publications {
+    release(MavenPublication) {
+      pom.withXml {
+        def root = asNode()
+        root.appendNode('name', 'App Engine Gradle Plugin')
```

Comment:
What do you mean? It's kind of a dumb part of using the maven-publish plugin. I don't know why they did it like this. But this is how you modify the pom.